ui/desktop/app/styles/app.scss

```diff
@@ -135,6 +135,22 @@
     margin: sizing.rems(xs);
   }
 
+  .button-window-close {
+    &:not(:disabled) {
```

Comment:
Any reason this was kept for the close button but not the minimize/fullscreen buttons?

Reply:
That was left in err. Removed.